### ISSUE TITLE
OSX: 1.4.0-rc.4: TCP connection fails with GUI

### ISSUE THREAD

**Issue by SynthiaPayne** [jacktrip/jacktrip#407]
on Catalina 10.15.7, using the GUI for RtAudio or JACK is not working for me: GUI - TCP Connection Timeout
Connects fine using the command line.

![Screen Shot 2021-09-15 at 6 03 51 PM](https://user-images.githubusercontent.com/2858155/133532745-86764088-ee32-4cc3-a417-9c0284434974.png)

**Comment by SynthiaPayne** [comment]
Tried downloads from the Github and nightly links

**Comment by SynthiaPayne** [comment]
This issue seems to have been resolved.

**Comment by psiborg112** [comment]
Need more info. Most likely reason is some sort of networking issue. If you try connecting using the IP address of the server rather than the host name, does it work? (You can find the IP by typing "host jackloop256.stanford.edu" into the command line.)

**Comment by SynthiaPayne** [comment]
Yes! I was able to connect using the IP address! Thanks! 
If I want to use an external interface without JACK, using Rt Audio it connects, but does not send the audio through the external. Is that another issue?

**Comment by SynthiaPayne** [comment]
I'll close this again because the issue was resolved by entering an IP address rather than the server alias name.

**Comment by psiborg112** [comment]
Yes, that would be a different issue. Worth checking to see why the DNS lookup isn't working though. Just to rule things out, is there any whitespace after the address?

**Comment by SynthiaPayne** [comment]
That was it! There was a blank space after the alias name, and when I took it off it connected perfectly. Good eye Aaron! 
Regarding using a USB mic with Rt Audio, I will add to the issue #413 about the Volume Controls not active when using a USB mic with Rt Audio (not Jack). Thanks!

**Comment by psiborg112** [comment]
Awesome. #412 trims any additional whitespace at the beginning or end of the address so will stop this from happening in future.
